src/Path/CheckInstall.hs

```diff
@@ -25,32 +27,29 @@ warnInstallSearchPathIssues destDir installed = do
                 Just exePath -> do
                     exeDir <- (liftIO . fmap FP.takeDirectory . D.canonicalizePath) exePath
                     unless (exeDir `FP.equalFilePath` destDir) $ do
-                        logWarn ""
-                        logWarn $ T.concat
-                            [ "WARNING: The \""
-                            , exe
-                            , "\" executable found on the PATH environment variable is "
-                            , T.pack exePath
-                            , ", and not the version that was just installed."
-                            ]
-                        logWarn $ T.concat
-                            [ "This means that \""
-                            , exe
-                            , "\" calls on the command line will not use this version."
-                            ]
+                        prettyWarnL
```

Comment:
Would be best to combine these two into a single warn call so that it just says "Warning:" once

Reply:
Ah, yes nice catch thanks. Will change.